packages/suite/src/actions/wallet/fiatRatesActions.ts

```diff
@@ -253,11 +253,8 @@ export const updateStaleRates = () => async (dispatch: Dispatch, _getState: GetS
 export const updateLastWeekRates = () => async (dispatch: Dispatch, getState: GetState) => {
     const { localCurrency } = getState().wallet.settings;
 
-    // unix timestamp in seconds - 3 mins (blockbook returns empty object if timestamp is too fresh)
-    const currentTimestamp = getBlockbookSafeTime();
-    const weekAgoTimestamp = currentTimestamp - 7 * 86400;
-
-    const timestamps = [weekAgoTimestamp, currentTimestamp];
+    const weekAgoTimestamp = getUnixTime(subWeeks(new Date(), 1));
+    const timestamps = [weekAgoTimestamp];
```

Comment:
you may be wondering why I am keeping it as an array. 2 reasons:
1. it has always been an array and I don't want to write db migrations 😓 
2. I heard some plans that in the future we will again start showing graph for exchange rate, so I don't want to write migrations twice! (one for array -> number second for number -> array)